guides/json.md

```diff
@@ -0,0 +1,118 @@
+- [Making Raw JSON REST Requests](#making-raw-json-rest-requests)
+  - [GET](#get)
+  - [PUT](#put)
+  - [POST](#post)
+  - [DELETE](#delete)
+
+# Making Raw JSON REST Requests
+
+The OpenSearch client implements many high-level REST DSLs that invoke OpenSearch APIs. However you may find yourself in a situation that requires you to invoke an API that is not supported by the client. Use `client.Perform` to do so.
+
+## GET
+
+The following example returns the server version information via `GET /`.
+
+```go
+infoRequest, err := http.NewRequest("GET", "/", nil)
```

Comment:
If you don´t want to check if the err is nil, you can omit it by using `_`. 
It would look like this:
```suggestion
infoRequest, _ := http.NewRequest("GET", "/", nil)
```
otherwise:
```suggestion
infoRequest, err := http.NewRequest("GET", "/", nil)
if err != nil {
    fmt.Printf("failed to perform request: %s", err)
    os.Exit(1)
}
```
This repeats for PUT,POST and DELETE.